hack/prebuild/local-dev/main.tf

```diff
@@ -0,0 +1,29 @@
+provider "libvirt" {
+  uri = "${var.libvirt_uri}"
+}
+
+resource "libvirt_network" "dev_net" {
+  name   = "${var.libvirt_network_name}"
+  mode   = "nat"
+  bridge = "${var.libvirt_network_if}"
+  domain = "${var.base_domain}"
+
+  addresses = [
+    "${var.libvirt_ip_range}",
+  ]
+
+  dns = {
+    local_only = true
+
+    forwarders = [{
+      address = "${var.libvirt_resolver}"
+    }]
+  }
+
+  autostart = true
+}
+
+resource "libvirt_volume" "rh_cos_base" {
+  name   = "rh_cos_base"
+  source = "${var.libvirt_base_image}"
+}
```

Comment:
we'll need:
- A default pool https://github.com/openshift/installer/blob/master/docs/dev/libvirt-howto.md#configure-default-libvirt-storage-pool
- The base volume
- Ignition  config
See https://github.com/openshift/machine-api-operator/blob/master/machines/libvirt/worker.machineset.yaml#L29

Reply:
@enxebre seems like the terraform libvirt plugin does not manage storage pools (https://github.com/dmacvicar/terraform-provider-libvirt/issues/76). I suggest to use a combination of bash/terraform. What do you think?

Reply:
How would that look like? Terraform running bash script to provision volumes?

Reply:
@paulfantom I'm not a fan of this solution either... but technically, yes, it can be done, if we just want to run a simple `terraform apply` in one go and not add one manual step.

Reply:
both approaches are valid. let's see if anyone else has any preference, one way or another.